### PR TITLE
[circle2circle] Revise driver script

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit revises driver script for static analysis tool.

Related: #13806
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/circle2circle/src/Circle2Circle.cpp [hunk 1]
```diff
@@ -262,164 +262,102 @@ int entry(int argc, char **argv)
...
+  option_str_to_enum["fuse_horizontal_fc_layers"] = Algorithms::FuseHorizontalFullyConnected;
+  option_str_to_enum["fuse_batchnorm_with_conv"] = Algorithms::FuseBatchNormWithConv;
+  option_str_to_enum["fuse_add_to_fullyconnected_bias"] = Algorithms::FuseAddToFullyConnectedBias;
+  option_str_to_enum["fuse_add_with_conv"] = Algorithms::FuseAddWithConv;
+  option_str_to_enum["fuse_add_with_fully_connected"] = Algorithms::FuseAddToFullyConnectedBias;
+  option_str_to_enum["fuse_add_with_tconv"] = Algorithms::FuseAddWithTConv;
+  option_str_to_enum["fuse_batchnorm_with_dwconv"] = Algorithms::FuseBatchNormWithDwConv;
+  option_str_to_enum["fuse_batchnorm_with_tconv"] = Algorithms::FuseBatchNormWithTConv;
+  option_str_to_enum["fuse_mul_to_fullyconnected_weights"] =
+    Algorithms::FuseMulToFullyConnectedWeights;
```
Comment (seanshpark) [inline compiler/circle2circle/src/Circle2Circle.cpp:290]: let's use `// clang-format off` and put these two to single line :)

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

File: compiler/circle2circle/src/Circle2Circle.cpp [hunk 2]
```diff
@@ -262,164 +262,97 @@ int entry(int argc, char **argv)
...
+  option_str_to_enum["fold_sparse_to_dense"] = Algorithms::FoldSparseToDense;
+  option_str_to_enum["fold_squeeze"] = Algorithms::FoldSqueeze;
+  option_str_to_enum["forward_reshape_to_unaryop"] = Algorithms::ForwardReshapeToUnaryOp;
+  option_str_to_enum["forward_transpose_op"] = Algorithms::ForwardTransposeOp;
+  option_str_to_enum["fuse_activation_function"] = Algorithms::FuseActivationFunction;
+  option_str_to_enum["fuse_horizontal_fc_layers"] = Algorithms::FuseHorizontalFullyConnected;
+  option_str_to_enum["fuse_batchnorm_with_conv"] = Algorithms::FuseBatchNormWithConv;
+  option_str_to_enum["fuse_add_to_fullyconnected_bias"] = Algorithms::FuseAddToFullyConnectedBias;
+  option_str_to_enum["fuse_add_with_conv"] = Algorithms::FuseAddWithConv;
+  option_str_to_enum["fuse_add_with_fully_connected"] = Algorithms::FuseAddToFullyConnectedBias;
```
Comment (ejjeong) [inline compiler/circle2circle/src/Circle2Circle.cpp:286]: Q: shouldn't it be like below?

I met a correctness issue from an internal quantized model after I updated `one-compiler`.
(FYI, e.g., `one-compiler=1.29.0~24082216` was fine.)

```diff
-  option_str_to_enum["fuse_add_with_fully_connected"] = Algorithms::FuseAddToFullyConnectedBias;
+  option_str_to_enum["fuse_add_with_fully_connected"] = Algorithms:: FuseAddWithFullyConnected;
```

https://github.com/Samsung/ONE/blob/daf9a85b154f7d67c7047d156a1018651d0d3000/compiler/one-cmds/how-to-use-one-commands.txt#L167
https://github.com/Samsung/ONE/blob/daf9a85b154f7d67c7047d156a1018651d0d3000/compiler/one-cmds/how-to-use-one-commands.txt#L169

Comment (seanshpark) [reply compiler/circle2circle/src/Circle2Circle.cpp:286]: @ejjeong , plz fire fix PR :)

**Pull request merged**